src/KeyVault/KeyVault/Commands/Certificate/ImportAzureKeyVaultCertificate.cs

```diff
@@ -193,13 +273,15 @@ public override void ExecuteCmdlet()
                         }
                         break;
 
-                    case ImportWithPrivateKeyFromCollectionParameterSet:
-                        certBundle = this.DataServiceClient.ImportCertificate(VaultName, Name, CertificateCollection, Tag?.ConvertToDictionary());
+                    case ImportWithPrivateKeyFromCollectionWithPolicyParameterSet:
+                    case ImportWithPrivateKeyFromCollectionWithPolicyFileParameterSet:
+                        certBundle = this.Track2DataClient.ImportCertificate(VaultName, Name, CertificateCollection, Password, Tag?.ConvertToDictionary(), certPolicy: policy);
```

Comment:
Please verify CertificateCollection with password whether works. I doubt this function can't support on windows.

Reply:
Verified.